recipes/frictionqpotfem/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "frictionqpotfem" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "01a42d2ff684669281bad5f004906b599d96eb5416aadad9e57a4ded7ce57799" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/FrictionQPotFEM/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+    - goosefem
+    - gmatelastoplasticqpot
+    - xtensor
```

Comment:
These are either:
* not needed at all (outside of `run`)
* belong in `host`

I'd suggest removing them and adding them as `host` deps if needed
```suggestion
```

Reply:
Thanks for your review @chrisburr . I need clarification of your remark: all these libraries are loaded by CMake during install. So I don't think `build` could work without them. Then the user always needs them when using the header-only library. But since the user only needs them at compile time I don't want to fix them to any version of this library (again this library is header-only, so nothing is actually compiled in the feedstock). So I still think that I did right...

Reply:
As this is header only I guess the "install" is basically (vastly over-simplifying):
```
cp myheader.h $PREFIX/include/myheader.h
```
In which case they're probably not actually needed, except I guess CMake is looking for them anyway even if they're not used? If so, they're needed for the "target" platform and this should be:
```yaml
  build:
    - {{ compiler('cxx') }}
    - cmake
    - make  # [unix]
  host:
    - goosefem
    - gmatelastoplasticqpot
    - xtensor
```
To try and explain the distinction: Imagine `xtensor` provides shared libraries and you're cross compiling for ARM from a `linux-64` machine. The compiler, `cmake` and `make` all need to be run-able on `linux-64` but they don't care about being executable on ARM. When you link to `libxtensor.so`, you need the ARM binary even though it can't be executed. Of course if they're header-only it probably doesn't make any difference if you use the ARM or x86 version of the package but it hypothetically could if you generate the headers differently for different platforms.

Reply:
Thanks @chrisburr ! 

Indeed installing is essentially copying the headers and generating the proper CMake config files. This is done with CMake that indeed looks for the dependencies. So at that time they need to be there. Furthermore, since the library is header only there is no linking involved. Only the end user will compile and use the dependencies (a side note is that all the listed dependencies are header-only libraries, thought for the discussion I don't think that matters). It does however need them all always when it want to compile with this library. So it does make sense to install them when the user installs the curent library. 

I think that the only way I can satisfy both requirements *without* fixing a version of this library to a specific version of the dependencies is to what I did. Note that this is an important requirement, as only the end user will compile, so only the end user will 'fix' the versions used.

Reply:
> I think that the only way I can satisfy both requirements without fixing a version of this library to a specific version of the dependencies is to what I did.

Are you looking for [`ignore_run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=ignore_run_exports#export-runtime-requirements) to make sure the `host` dependencies don't add version constraints at runtime?

Reply:
I have changed the recipe according to your suggestion